ilastik/workflows/tracking/conservation/conservationTrackingWorkflow.py

```diff
@@ -391,9 +391,9 @@ def post_process_lane_export(self, lane_index, time=1):
                     logger.error("Cannot export from plugin with empty output filename")
                     return
 
-                if time == 0 and selectedPlugin == 'H5-Event-Sequence' and os.path.exists(filename + '/H5-Event-Sequence/'):
+                if overwriteExistingFiles == False and selectedPlugin == 'H5-Event-Sequence' and os.path.exists(filename + '/H5-Event-Sequence/'):
```

Comment:
@chaubold thanks for the idea, I hope my last commit implements it the way you thought of it.